tensorflow_addons/activations/tanhshrink_test.py

```diff
@@ -18,30 +18,51 @@
 import numpy as np
 import tensorflow as tf
 from tensorflow_addons.activations import tanhshrink
+from tensorflow_addons.activations.tanhshrink import _tanhshrink_py
 from tensorflow_addons.utils import test_utils
 
 
 @test_utils.run_all_in_graph_and_eager_modes
 class TanhshrinkTest(tf.test.TestCase, parameterized.TestCase):
+    # @parameterized.named_parameters(
+    #     ("float16", np.float16), ("float32", np.float32), ("float64", np.float64)
+    # )
+    # def test_tanhshrink(self, dtype):
+    #     x = tf.constant([-2.0, -1.0, 0.0, 1.0, 2.0], dtype=dtype)
+    #     expected_result = tf.constant(
+    #         [-1.0359724, -0.23840582, 0.0, 0.23840582, 1.0359724], dtype=dtype
+    #     )
+
+    #     self.assertAllCloseAccordingToType(tanhshrink(x), expected_result)
+
+    # @parameterized.named_parameters(("float32", np.float32), ("float64", np.float64))
+    # def test_theoretical_gradients(self, dtype):
+    #     # Only test theoretical gradients for float32 and float64
+    #     # because of the instability of float16 while computing jacobian
+    #     x = tf.constant([-2.0, -1.0, 0.0, 1.0, 2.0], dtype=dtype)
+
+    #     theoretical, numerical = tf.test.compute_gradient(tanhshrink, [x])
+    #     self.assertAllCloseAccordingToType(theoretical, numerical, atol=1e-4)
+
```

Comment:
Maybe we could actually remove all that? The check with random tensors is much more robust. What do you think?

Reply:
Good idea. I remove them later.